docs/02-skog-context.md

```diff
@@ -0,0 +1,111 @@
+# Async Context in Skog
+
+In this page you will learn more on how Skog uses Async Context and how the functions `runWithSkogContext` and `getFields` can help you to make integrations using its power.
+
+- Both functions handle **Node.js Async Context**. If you don't know what it is, you can read about it [here](./01-nodejs-context.md).
+- This page is not the API reference of the functions but an explanation of them. [ todo: create such page and add a link to it here ]
+
+---
+
+- The function **runWithContext** receives two arguments: `fields` (an object) and `callback` (a zero-argument function).
+
+  It will create a new context with `fields`. The context will be alive during the execution of the `callback`.
+
+  It will return the same value as the value returned by the callback.
+
+- The function **getFields** returns the `fields` in the current execution context. Will return null if no fields are set or if called outside of an execution context.
```

Comment:
It is not obvious how Skog uses getFields. Will it add them to the logging object?

Reply:
- Added an explanation about how `getFields` is called internally
- Added also `getFields` in the examples (which is what the actual implementation does)